.github/ISSUE_TEMPLATE/d--keras-preprocessing-users.md

```diff
@@ -0,0 +1,12 @@
+---
+name: a) `keras.preprocessing` users
```

Comment:
I believe it should be "d" if we do this (following the pattern of the other files).